pulsar-broker/src/main/java/org/apache/pulsar/broker/service/persistent/PersistentTopic.java

```diff
@@ -1396,6 +1423,7 @@ private CompletableFuture<Void> delete(boolean failIfHasSubscriptions,
                                         unfenceTopicToResume();
                                         deleteFuture.completeExceptionally(ex);
                                     } else {
+                                        log.info("[{}] Topic deleted...", topic);
```

Comment:
I believe these logs are only added for debugging. It's better to delete them.

Reply:
Yes, remove them.